frontend/src/components/maps/leaflet/InfoSlideOut/InfoSlideOut.tsx

```diff
@@ -123,10 +149,17 @@ const InfoControl: React.FC<InfoControlProps> = ({ open, setOpen, onHeaderAction
       L.DomEvent.on(elem!, 'mousewheel', L.DomEvent.stopPropagation);
     }
   });
+
+  //whether the associated buildings info is open
+  const [buildingsOpen, setBuildingsOpen] = React.useState<boolean>(false);
+  //whether the general parcel info is open
+  const [parcelInfoOpen, setParcelInfoOpen] = React.useState<boolean>(true);
+
   const addAssociatedBuildingLink = (
     <>
       <FaPlusSquare color="#1a5a96" className="mr-1" />
       <Link
+        style={{ color: '#1a5a96' }}
```

Comment:
do we know if there's anyway to use variable names here?

Reply:
should be just able to use the variable name directly no ' or ` required.

Reply:
since I'm pretty sure variables.slideOutBlue is just a string.